data/json/items/gun/robofac_gun.json

```diff
@@ -506,9 +506,9 @@
     "type": "MAGAZINE",
     "name": { "str": "HWP 60-round 12.3 magazine" },
     "description": "A box magazine for use with the Hub 01 Hybrid Weaons Platform, developed for the 12.3 ln round.  Physically speaking, this looks like one of the box magazines that the Exodii use for their own guns, but with a more ergonomic design afforded by the Hub's use of modern materials.",
```

Comment:
> Physically speaking, this looks like one of the box magazines that the Exodii use for their own guns

Perhaps something like ` It resembles a coffin somewhat in profile, with 4 columns of ammunition tapering down to just 2.`

Also should fix the typo of 'weaons' while you're at it.